packages/renderer/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Workspace, WorkspaceConfig } from '@capsulajs/capsulahub-core-workspace/lib/api';
+import * as api from './api';
+
+declare global {
+  interface Window {
+    CAPSULAHUB_WORKSPACE: Workspace;
+    CAPSULAHUB_CONFIG: WorkspaceConfig;
```

Comment:
`CAPSULAHUB_CONFIG` won't be all the workspace config, it will be only the Renderer config. We need to define this config interface (if it exists or have an empty interface) and put it here.